backend/src/api/routes/oauth-apps/index.routes.ts

```
@@ -0,0 +1,89 @@
+    successResponse(res, { success: true });
+  } catch (error) {
+    next(error);
+  }
+});
+
+/**
+ * AUTHORIZED APPS (Project-level)
+ */
+
+// GET /api/oauth-apps/authorizations/:projectId
+router.get('/authorizations/:projectId', verifyAdmin, async (req: AuthRequest, res: Response, next: NextFunction) => {
```

ESLint

[error] Replace `'/authorizations/:projectId',·verifyAdmin,` with `⏎··'/authorizations/:projectId',⏎··verifyAdmin,⏎·`

(prettier/prettier)